src/types.ts

```diff
@@ -2770,7 +2808,7 @@ export type DrawTextOpt = RenderProps & {
 	/**
 	 * The name of font to use.
 	 */
-	font?: string,
+	font?: string | FontData,
```

Comment:
unrelated, but nice to also enable passing in raw font data from `loadFont`, also makes the type more compatible with the internal version and the exposed version